#! /bin/sh
patch -p1 -f $* < $0
exit $?

2005-11-17  Werner Koch  <wk@g10code.com>

	* GNUmakefile (install): Add new target.


--- pthreads-w32-2-7-0-release.orig/GNUmakefile	2005-06-03 10:36:27.000000000 +0200
+++ pthreads-w32-2-7-0-release/GNUmakefile	2005-11-17 10:35:41.000000000 +0100
@@ -541,6 +541,22 @@
 	$(RANLIB) $(GC_LIB)
 	echo touched > $(GC_STATIC_STAMP)
 
+
+install:
+	set -e; \
+        for i in include lib bin ; do \
+           mkdir -p "$(DESTDIR)$(prefix)/$$i" || true ; \
+        done; \
+        for i in pthread.h sched.h semaphore.h ; do \
+           cp "$$i" "$(DESTDIR)$(prefix)/include/$$i" ; \
+        done; \
+        for i in libpthread*.a pthread.def ; do \
+           cp "$$i" "$(DESTDIR)$(prefix)/lib/$$i" ; \
+        done; \
+        for i in pthread*.dll ; do \
+           cp "$$i" "$(DESTDIR)$(prefix)/bin/$$i" ; \
+        done
+
 clean:
 	-$(RM) *~
 	-$(RM) *.i
